docs/react/start-streaming.mdx

```diff
@@ -31,26 +32,35 @@ To manage users' camera and microphone devices, use the respective `useCamera` a
 Both of them has the same API. To keep things simple, let's just use the camera hook.
 
 ```tsx
+import { useEffect, useRef } from "react";
 import { useCamera } from "@fishjam-cloud/react-client";
 
-function ExampleComponent() {
-  const camera = useCamera();
+export function ExampleComponent() {
+  const videoRef = useRef<HTMLVideoElement>(null);
 
-  return (
-    <div>
-      {camera.isStreaming && <p>You are going live!</p>}
+  // highlight-next-line
+  const { isStreaming, activeDevice, initialize, stream, devices } =
+    // highlight-next-line
+    useCamera();
```

Comment:
**question:** 
```suggestion
  const { isStreaming, activeDevice, initialize, stream, devices } =     useCamera();
    // highlight-next-line
```
was it intended to put `useCamera` in another line?

Reply:
it was autoformatted :(